nipype/interfaces/ants/segmentation.py

```diff
@@ -1496,75 +1394,91 @@ class AntsJointFusionInputSpec(ANTSCommandInputSpec):
     verbose = traits.Bool(False, argstr="-v", desc=("Verbose output."))
 
 
-class AntsJointFusionOutputSpec(TraitedSpec):
+class _JointFusionOutputSpec(TraitedSpec):
     out_label_fusion = File(exists=True)
-    out_intensity_fusion_name_format = traits.Str()
-    out_label_post_prob_name_format = traits.Str()
-    out_atlas_voting_weight_name_format = traits.Str()
+    out_intensity_fusion = OutputMultiPath(File(exists=True))
+    out_label_post_prob = OutputMultiPath(File(exists=True))
+    out_atlas_voting_weight = OutputMultiPath(File(exists=True))
 
 
-class AntsJointFusion(ANTSCommand):
+class JointFusion(ANTSCommand):
```

Comment:
why did this get renamed? i'm pretty positive mindboggle uses this interface.

Reply:
Yup, I guess the Ants prefix was added not to clobber the old JointFusion. Now that the legacy binary is not supported, we can normalize the naming w.r.t. to the rest of the ants module.

As you found below, I added an alias for backward compatibility.